testutil/sims/app_helpers.go

```diff
@@ -275,7 +275,7 @@ func GenesisStateWithValSet(
 	// add bonded amount to bonded pool module account
 	balances = append(balances, banktypes.Balance{
 		Address: authtypes.NewModuleAddress(stakingtypes.BondedPoolName).String(),
-		Coins:   sdk.Coins{sdk.NewCoin(sdk.DefaultBondDenom, bondAmt)},
+		Coins:   sdk.Coins{sdk.NewCoin(sdk.DefaultBondDenom, bondAmt.MulRaw(int64(len(delegations))))},
```

Comment:
Seems this was a bug in the Cosmos SDK right?

Reply:
Yep https://github.com/cosmos/cosmos-sdk/pull/23072